src/main/java/dev/jbang/cli/Export.java

```diff
@@ -660,6 +660,20 @@ void renderBuildFile(BuildContext ctx, Path projectDir, String fullClassName) th
 		boolean isGroovy = ctx.getProject().getMainSource() instanceof GroovySource;
 		boolean isKotlin = ctx.getProject().getMainSource() instanceof KotlinSource;
 		String kotlinVersion = isKotlin ? ((KotlinSource) ctx.getProject().getMainSource()).getKotlinVersion() : "";
+		StringBuilder jvmArgs = new StringBuilder();
+		for (String arg : ctx.getProject().getRuntimeOptions()) {
+			if (jvmArgs.length() > 0) {
+				jvmArgs.append(", ");
+			}
+			jvmArgs.append(String.format("'%s'", arg));
+		}
+		StringBuilder compilerArgs = new StringBuilder();
+		for (String arg : ctx.getProject().getMainSourceSet().getCompileOptions()) {
```

Comment:
Looks good in general, but perhaps you could introduce a method for this to avoid the code duplication?

Reply:
Created method gradleArgs(List<String> options)